gs-web-elasticsearch/doc/index.rst

```diff
@@ -296,60 +296,170 @@ Grid Strategy
 
-* ``basic``: returns top level ``doc_count``.
+Basic
+~~~~~
+Returns top level ``doc_count``.
 
-    Example Aggregation::
+Example Aggregation::
+
+  {
+    "agg": {
+      "geohash_grid": {
+        "field": "geo"
+      }
+    }
+  }
     
-      {
-        "agg": {
-          "geohash_grid": {
-            "field": "geo"
+Example bucket::
+
+ {
+   "key" : "xv",
+   "doc_count" : 1
+ }
+
+Extracted raster value: ``1``
+
+Metric
+~~~~~~
+Returns metric value.
+
+.. list-table::
+   :widths: 20 20 60
+
+   * - Parameter
+     - Default Value
+     - Description
+   * - Metric key
+     - ``metric``
+     - Key used to pluck metric object from top level bucket. Empty string results in plucking doc_count.
+   * - Value key
+     - ``value``
+     - Key used to pluck the value from the metric object.
+
+Example Aggregation::
+
+  {
+    "agg": {
+      "geohash_grid": {
+        "field": "geo"
+      },
+      "aggs": {
+        "metric": {
+          "max": {
+            "field": "magnitude"
           }
         }
       }
+    }
+  }
+
+Example bucket::
+
+  {
+    "key" : "xv",
+    "doc_count" : 1,
+    "metric" : {
+      "value" : 4.9
+    }
+  }
     
-   Example bucket::
- 
-     {
-       "key" : "xv",
-       "doc_count" : 1
-     }
-  
-  Raster value: ``1``
-
-* ``metric``: returns metric value.
-
-    Example Aggregation::
-    
-      {
-        "agg": {
-          "geohash_grid": {
-            "field": "geo"
-          },
-          "aggs": {
-            "metric": {
-              "max": {
-                "field": "magnitude"
-              }
-            }
+Extracted raster value: ``4.9``
+
+Nested_agg
+~~~~~~~~~~
+Extract raster value from nested aggregation results.
+
+.. list-table::
+   :widths: 20 20 60
+
+   * - Parameter
+     - Default Value
+     - Description
+   * - Aggregation key
+     - ``nested``
+     - Key used to pluck nested aggregation results from the geogrid bucket.
+   * - Metric key
+     - empty string
+     - Key used to pluck metric object from each nested aggregation bucket. Empty string results in plucking doc_count.
+   * - Value key
+     - ``value``
+     - Key used to pluck the value from the metric object.
+   * - Selection strategy
+     - ``largest``
+     - ``largest`` | ``smallest``. Strategy used to select a bucket from the nested aggregation buckets. The grid cell raster value is extracted from the selected bucket.
+   * - Raster strategy
+     - ``value``
+     - ``key`` | ``value``. Strategy used to extract the raster value from the selected bucket. ``value``: Raster value is the selected bucket's metric value. ``key``: Raster value is the selected bucket's key.
+   * - Token map
+     - null
+     - Map used to convert String keys into numeric values. Use the format ``key1:1;key2:2``. Only utilized when raster strategy is ``key``.

```

Comment:
And same question as above regarding more clearly identifying these parameters as elements in the `gridStrategyArgs` list.